packages/wrangler/src/type-generation/runtime/index.ts

```diff
@@ -55,7 +71,7 @@ export async function generateRuntimeTypes({
 		),
 	});
 
-	await writeFile(outFile, types, "utf8");
+	await writeFile(outFile, header + "\n" + types, "utf8");
```

Comment:
```suggestion
	await writeFile(outFile, `${header}\n${types}`, "utf8");
```
I thought we had an ESLint rule for this?